README.md

```diff
@@ -1,13 +1,13 @@
-# Prometeo sensor simulator
+# Pyrrha sensor simulator
 
-This repository will contain the [Prometeo](https://github.com/Call-for-Code/Prometeo) solution sensor simulator.
+This repository will contain the [Pyrrha](https://github.com/Call-for-Code/Prometeo) solution sensor simulator.
```

Comment:
Need to update the URL too

Reply:
https://github.com/Pyrrha-Platform/Pyrrha 
going to go with this one?

Reply:
Yes please

Reply:
url has been updated